packages/block-library/src/navigation/edit/index.js

```diff
@@ -310,6 +328,42 @@ function Navigation( {
 	] = useState();
 	const [ detectedOverlayColor, setDetectedOverlayColor ] = useState();
 
+	const [
+		showClassicMenuConversionErrorNotice,
+		hideClassicMenuConversionErrorNotice,
+	] = useNavigationNotice( {
+		name: 'block-library/core/navigation/classic-menu-conversion/error',
+	} );
+
+	function handleUpdateMenu( menuId ) {
+		setRef( menuId );
+		selectBlock( clientId );
+	}
+
+	useEffect( () => {
+		if ( classicMenuConversionStatus === 'pending' ) {
```

Comment:
We use `"pending"` here and `CLASSIC_MENU_CONVERSION_PENDING` in the other file – how about using these constants here as well? I don't like that long name, though. Ideally we could rely on TS to verify the right-hand side here.

Reply:
Yeh I was thinking that as well 😄 

AFAIK we aren't supposed to use TS at this level (Gutenberg informal agreement) but yes it would be ideal.

Reply:
✅